implementations/rust/ockam/ockam_api/src/cloud/project/models.rs

```diff
@@ -0,0 +1,296 @@
+use crate::cloud::addon::KafkaConfig;
+use crate::cloud::email_address::EmailAddress;
+use crate::cloud::share::{RoleInShare, ShareScope};
+use crate::minicbor_url::Url;
+use minicbor::{Decode, Encode};
+use ockam::identity::Identifier;
+use serde::{Deserialize, Serialize};
+use std::fmt::{Display, Formatter};
+
+#[derive(Encode, Decode, Serialize, Deserialize, Debug, Default, Clone, Eq, PartialEq)]
+#[cbor(map)]
+pub struct ProjectModel {
```

Comment:
As we get closer to CQRS/DDD, perhaps we could start introducing more concepts to disambiguate between domain and application layers, especially in this crate. Maybe this could be named `ProjectViewModel`?

Reply:
Could you please elaborate why it deserves `View` part? After this refactoring the only one responsibility of this struct is to parse Controller's response.

Reply:
This is subjective and open to discussion, but "model" feels too generic, and in some paradigms a model can contain logic. The "view" part can enforce that this model is just data with no logic.